scripts/set-defaults.sh

```diff
@@ -0,0 +1,29 @@
+# Check variables below and fix their value to default one if they aren't set or their value is empty string. 
```

Comment:
Should we add `#!/bin/bash -e` to the top of this file? like we have for all other Bash scripts

Reply:
no, as it supposed to be executed via `source` that will ignore shebang and bash parameters. 
Quote from `source --help`:

>     Exit Status:
    Returns the status of the last command executed in FILENAME; fails if
    FILENAME cannot be read.